src/main/java/dev/slimevr/bridge/ProtobufBridge.java

```diff
@@ -177,6 +177,9 @@ protected void userActionReceived(UserAction userAction) {
 				// TODO : Check pose field
 				Main.vrServer.resetTrackers();
 				break;
+			case "fast reset":
```

Comment:
The naming doesn't make sense. You call reset "calibrate", and fast reset - "fast reset". Please update naming to match, and I also don't think that using space is a good idea in the name, maybe use underscore `_`?

Reply:
Understandable. I didn't change calibrate to reset for backwards compatibility reasons. Perhaps I can make reset trigger for both "calibrate" and "reset", and mark accepting "calibrate" as deprecated?

I'll change "fast reset" to "fast_reset". That said, whitespace *should* be fine here.

Reply:
I didn't mean it will cause problems, just don't think spaces are good in such places conceptually. Also yes, adding both as an option with one deprecated is fine!